tests/components/sunsynkweb/test_coordinator.py

```diff
@@ -0,0 +1,59 @@
+"""Basic coordinator tests."""
```

Comment:
don't test the internals (coordinator) but the effects of the exceptions in the coordinator (e.g. entities going unavailable)

Reply:
you are still testing the coordinator